apps/google-analytics-4/lambda/src/controllers/concerns/error-class-to-api-error-map.ts

```diff
@@ -0,0 +1,27 @@
+// Maps an error class name -> a handler function that takes the error of that type as input and returns
```

Comment:
Just the minorest of nitpick comments:

Not sure about introducing a "Rails" concept name here 😅 . I feel like it ends up feeling awkward or weird to folks who aren't familiar with that ecosystem.

I wish I had a better idea for the name / location of this bit of logic. It's definitely application specific -- it's our own mapping that's aware of multiple aspects of the code base (the error middleware, specific error types our controllers emit, how we want the server to respond).

For now i guess I don't have a better idea where this could go!

Reply:
moving to middlewares 👍

Reply:
actually i just deleted this file and moved it as a const into the apiErrorHandler middleware ✅